basin3d/plugins/usgs.py

```diff
@@ -224,50 +240,49 @@ def _load_point_obj(datasource, json_obj, feature_observed_properties, synthesis
     #  huc_cd          -- Hydrologic unit code
 
     :param json_obj:
-    :param feature_observed_properties: dictionary of locations and their available variables
-    :type feature_observed_properties: dict
+    :param observed_property_variables: list of locations and their available variables
+    :param synthesis_messages
     :return:
     """
 
     if 'site_no' in json_obj:
         id = json_obj['site_no']
 
-        # Get the location variables from the dictionary
-        if observed_property_variables is None:
-            if id in feature_observed_properties.keys():
-                observed_property_variables = feature_observed_properties[id]
-
         lat, lon = None, None
         try:
             lat, lon = float(json_obj['dec_lat_va']), float(json_obj['dec_long_va'])
         except Exception as e:
             synthesis_messages.append(f"Error getting latlon: {str(e)}")
             logger.error(str(e))
-
-        monitoring_feature = MonitoringFeature(
-            datasource,
-            id="{}".format(id),
-            name=json_obj['station_nm'],
-            feature_type=FeatureTypeEnum.POINT,
-            shape=SpatialSamplingShapes.SHAPE_POINT,
-            related_sampling_feature_complex=[RelatedSamplingFeature(
+        if id not in observed_property_variables.keys():
+            id = "Could not find observed property variables for this monitoring feature."
+            synthesis_messages.append(f"Could not find observed property variables for this monitoring feature: {id}")
+            logger.error({id})
+        else:
```

Comment:
We want the ability to build a monitoring feature object even if we can't find the observed property values.

Please detach the error handling from the modeling object build. Also, please don't replace the id with the error message. Make it a warning instead of an error.

Something along the lines of the following:

```
         if id not in observed_property_variables.keys():
             mf_opv = "Could not find observed property variables for this monitoring feature."
             msg = f"Could not find observed property variables for this monitoring feature: {id}"
             synthesis_messages.append(msg)
             logger.warning({msg})
         else:
             mf_opv = observed_property_variables[id]
```

Then use `mf_opv` in new line 276.